guides/common/modules/proc_synchronizing-the-satellite-tools-repository.adoc

```diff
@@ -51,7 +52,8 @@ For more information,
 see https://www.redhat.com/en/resources/els-datasheet[{RHEL} Extended Life Cycle Support (ELS) Add-on] guide.
 ====
 
-To use the CLI instead of the {ProjectWebUI}, see the xref:CLI_Synchronizing_the_Client_Repository_rhel_7_{context}[] or xref:CLI_Synchronizing_the_Client_Repository_rhel_6_{context}[].
+To use the CLI instead of the {ProjectWebUI}, see the xref:CLI_Synchronizing_the_Client_Repository_rhel_7_{context}[Synchronizing the Client Repository on RHEL 7] or 
+xref:CLI_Synchronizing_the_Client_Repository_rhel_6_{context}[Synchronizing the Client Repository on RHEL 6].
```

Comment:
```suggestion
To use the CLI instead of the {ProjectWebUI}, see the procedure relevant for your {RHEL} version:
* xref:CLI_Synchronizing_the_Client_Repository_rhel_7_{context}[]
* xref:CLI_Synchronizing_the_Client_Repository_rhel_6_{context}[]
```